inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -136,6 +136,7 @@
             </div>
             <div class="modal-body">
             <h4>INSPIRE keywords</h4>
+            
```

Comment:
Watch out! The stuff above this line doesn't belong to this PR.

Reply:
@Dinika please look at this. The above changes are already in `master` branch, so rebase and they should go away.